app/controllers/periods_controller.rb

```diff
@@ -25,6 +25,7 @@ def create
 
     respond_to do |format|
       if @period.save
+        # create_reports_for_bench_users(@period)
```

Comment:
This is safe to remove now that you have a callback in the Period model. If you wanted to keep the comment to refer to this version of the code, you still have the Git commit you can always jump back to

https://github.com/WilsonReese-AppDev/status-reporting/blob/f2b201d1a57436e4f2a3770c825b27bee58a9bfb/app/controllers/periods_controller.rb#L23-L36